Automattic/woocommerce-payments
# Dev: update @wordpress/scripts package to use dart-sass instead of node-sass

#### Changes proposed in this Pull Request

This PR updates wordpress/scripts version from 12.4.0 to 12.6.0 to switch from the deprecated node-sass to dart-sass to build stylesheets.

#### How I updated the files:

I ran `npm install @wordpress/scripts@12.6.0 --save-dev` to update both the package.json and package-lock.json files.







#### Testing instructions




* Run `npm ls node-sass` to check which packages use the `node-sass` package. 
* Checkout this branch and run `npm ci` which will update the packages according to the lock file. 
* Run `npm ls node-sass` to see if the packages are still there. There shouldn't be any.
* Run `npm ls sass` to see if the packages on the first step moved to use `sass` from `node-sass`
* Run `npm run build:client` to see if the client code still gets generated.

-------------------


**Post merge**
